sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -194,16 +212,16 @@ case class AdaptiveSparkPlanExec(
   }
 
   private def applyQueryPostPlannerStrategyRules(plan: SparkPlan): SparkPlan = {
-    applyPhysicalRules(
+    applyPhysicalRulesWithRuleContext(
       plan,
       context.session.sessionState.adaptiveRulesHolder.queryPostPlannerStrategyRules,
-      Some((planChangeLogger, "AQE Query Post Planner Strategy Rules"))
-    )
+      Some((planChangeLogger, "AQE Query Post Planner Strategy Rules")))
```

Comment:
In this case, let's recover to the old existing two lines in order to reduce the PR code diff.

Reply:
yeah, I think it's a typo when I do rebase